developer_manual/digging_deeper/profile.rst

```diff
@@ -0,0 +1,185 @@
+=======
+Profile
+=======
+
+The user profile presents the information of a user including their full name,
+profile picture, organisation, role, location, biography, headline, as well as
+actionable information that we call profile actions. These actions can include
+starting a `Nextcloud Talk <https://nextcloud.com/talk/>` chat with the user,
```

Comment:
Is it displayed well?
![obraz](https://user-images.githubusercontent.com/47037905/151509636-dfbb4b5d-ce0f-4570-9900-0985e17c7478.png)